### PR TITLE
Add homebrew tip

### PR DESCRIPTION
**Pull request by pstreef**

### CODE REVIEW

File: docs/moderne-cli/README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ Boot 2 to Spring Boot 3.
    </TabItem>
    </Tabs>
 
+  :::tip
+  On MacOS you can also use homebrew: `brew install moderneinc/moderne/mod` and skip step 4.
+  :::
+
```
Comment (timtebeek) [inline docs/moderne-cli/README.md:49]: Thanks for adding this! Can we move this into the Mac tab above? Then you can also drop the "On MacOS " prefix.

File: docs/moderne-cli/README.md [hunk 2]
```diff
@@ -43,13 +47,9 @@ Boot 2 to Spring Boot 3.
    </TabItem>
    </Tabs>
 
-  :::tip
-  On MacOS you can also use homebrew: `brew install moderneinc/moderne/mod` and skip step 4.
-  :::
-
-3. Once you have it downloaded, save it somewhere that your terminal can access. If you want it available in each of your terminal windows, consider updating your `PATH` to point to this location or aliasing `mod` to the location of the CLI. You could also save the file to a directory that's already on your `PATH` such as a `/usr/bin` directory.
+4. Once you have it downloaded, save it somewhere that your terminal can access. If you want it available in each of your terminal windows, consider updating your `PATH` to point to this location or aliasing `mod` to the location of the CLI. You could also save the file to a directory that's already on your `PATH` such as a `/usr/bin` directory.
```
Comment (timtebeek) [inline docs/moderne-cli/README.md]: Why 4 ?
```suggestion
3. Once you have it downloaded, save it somewhere that your terminal can access. If you want it available in each of your terminal windows, consider updating your `PATH` to point to this location or aliasing `mod` to the location of the CLI. You could also save the file to a directory that's already on your `PATH` such as a `/usr/bin` directory.
```

File: docs/moderne-cli/README.md [hunk 3]
```diff
@@ -43,13 +47,9 @@ Boot 2 to Spring Boot 3.
...
 
-4. Ensure you can run the CLI by typing `mod`.
+5. Ensure you can run the CLI by typing `mod`.
```
Comment (timtebeek) [inline docs/moderne-cli/README.md]: Why 5?
```suggestion
4. Ensure you can run the CLI by typing `mod`.
```

File: docs/moderne-cli/README.md (hunk 2 above)
Comment (pstreef) [reply docs/moderne-cli/README.md]: Already reverted, for some reason github does this automatically when I save through "edit a file"

**Review by timtebeek — APPROVED** [review]

**Pull request merged**